src/main/resources/modules/heart/pci_stent.json

```diff
@@ -0,0 +1,323 @@
+{
+  "name": "pci_stent",
```

Comment:
new file pci_stent.json with stent codes which map to icd10-pcs codes we are interested in